pyta_server/routes.py

```diff
@@ -0,0 +1,32 @@
+import os
+import secrets
+import psycopg2
+from flask import request
+from pyta_server import app, db
+from pyta_server.models import Uploads
+
+@app.route('/', methods=['POST'])
+def receive():
+    if request.method == 'POST':
+        dirs = {'source':'', 'config':''}
+        rand_hex = secrets.token_hex(8)
+        for f_type in request.files.keys():
```

Comment:
Don't do this as a loop. Instead, explicitly check for just `'source'` and `'config'` in the dictionary.